src/Storages/ObjectStorageQueue/ObjectStorageQueueTableMetadata.cpp

```diff
@@ -181,72 +184,17 @@ void ObjectStorageQueueTableMetadata::checkImmutableFieldsEquals(const ObjectSto
                 ErrorCodes::METADATA_MISMATCH,
                 "Existing table metadata in ZooKeeper differs in processing buckets. "
                 "Stored in ZooKeeper: {}, local: {}",
-                ObjectStorageQueueMetadata::getBucketsNum(*this), ObjectStorageQueueMetadata::getBucketsNum(from_zk));
+                ObjectStorageQueueMetadata::getBucketsNum(from_zk), ObjectStorageQueueMetadata::getBucketsNum(*this));
         }
     }
-}
-
-void ObjectStorageQueueTableMetadata::checkEquals(const ObjectStorageQueueSettings & current, const ObjectStorageQueueSettings & expected)
```

Comment:
This method is removed because it is now longer used, but we use `ObjectStorageQueueTableMetadata::checkEquals` instead there (it was annoying to have duplicate methods) (it was used in `ObejctStorageQueueMetadataFactory`, where now there is `checkEquals`)